celery/worker/consumer/consumer.py

```diff
@@ -379,6 +379,7 @@ def on_connection_error_after_connected(self, exc):
         try:
             self.connection.collect()
         except Exception:  # pylint: disable=broad-except
+            error('Error collecting connection: %r', exc, exc_info=True)
```

Comment:
What does error() do?

Reply:
its variable for logger.error. It is generally used at several places in celery code.